ansible/files/var/www/html/index.html

```diff
@@ -44,6 +44,8 @@ <h3>How to subscribe</h3>
 --
 subscribe
         </pre>
+
+        <p>* You can get the mail template by clicking the "Subscribe" button in the list and can send it as it is.</p>
```

Comment:
```suggestion
        <p>* You can get the mail template by clicking the "Subscribe" link in the list and can send it as it is.</p>
```

Reply:
Thank you for the suggestion.
I have fixed this.